frontend/src/lib/api.mock.ts

```diff
@@ -73,6 +73,7 @@ export const MOCK_DEFAULT_TEAM: TeamType = {
     session_recording_minimum_duration_milliseconds: null,
     session_recording_linked_flag: null,
     session_recording_network_payload_capture_config: null,
+    session_recording_summary_config: null,
```

Comment:
I made a `session_recording_config` in https://github.com/PostHog/posthog/pull/19583/files#diff-18b11537782452794a7ca49f8e734fc5c42477e6da0733ee052f2333412cfd6fR76 because we seemed to be adding a ton of different columns to the teams table. I wonder if we should use that here or possibly create a separate table for all these JSON columns? Probably good to align before we merge either of these PRs